docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "2"
```

Comment:
This is unrelated to Django :)

Reply:
I know, but I forgot to change it in the last PR and addes it here